osu.Game/Localisation/EditorSetupRulesetStrings.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Localisation;
+
+namespace osu.Game.Localisation
+{
+    public static class EditorSetupRulesetStrings
```

Comment:
I dunno how I feel about having like 10 files with single strings like this. It makes a translator's life hard because they have to go through many files.

Probably combine all of these into a single `EditorSetupStrings.cs`

Reply:
I considered it, but I felt the file would be too big, but I can combine it.

Reply:
It's hard to say, as new elements may be added over time. But files with single strings are also quite weird.